Compilation errors

```
/php-spx/src/php_spx.c: In function ‘zif_spx_profiler_start’:
/php-spx/src/php_spx.c:325:5: error: ‘for’ loop initial declarations are only allowed in C99 mode
     for (size_t i = 0; i < context.profiling_handler.depth; i++) {
     ^
php-spx/src/php_spx.c:325:5: note: use option -std=c99 or -std=gnu99 to compile your code
```

--- a/src/php_spx.c
+++ b/src/php_spx.c
@@ -322,7 +322,8 @@ static PHP_FUNCTION(spx_profiler_start)
         return;
     }
 
-    for (size_t i = 0; i < context.profiling_handler.depth; i++) {
+    size_t i;
+    for (i = 0; i < context.profiling_handler.depth; i++) {
         context.profiling_handler.profiler->call_start(
             context.profiling_handler.profiler,
             &context.profiling_handler.stack[i]

